common/rakelib/output.rake

```diff
@@ -0,0 +1,97 @@
+# Used constants:
+# none
+
+if File.directory? 'Tests/Expected'
```

Comment:
As you suggested in Slack, might not be worth sharing these tasks which are only used by the `templates` repo, so better move them directly in the `templates` repo's `Rakefile`.